WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40383,918 +40424,371 @@ e_latin_empire = {
 	}
 }
 
-d_miaphysite = {
```

Comment:
How is it messed up?

![new bitmap image](https://user-images.githubusercontent.com/17663470/28176753-87572792-67c6-11e7-9877-c14617d6fcef.png)

It works perfectly on my end.